packages/admin/dashboard/src/i18n/translations/fr.json

```
@@ -0,0 +1,2748 @@
+				"focusToolbar": "Focus barre d'outils",
+				"focusCancel": "Annuler le focus"
+			}
+		},
+		"errors": {
+			"fixError": "Fix error",
+			"count_one": "{{count}} erreur",
+			"count_other": "{{count}} erreurs"
+		}
+	},
+	"filters": {
+		"date": {
+			"today": "Aujourd'hui",
+			"lastSevenDays": "7 derniers jours",
+			"lastThirtyDays": "30 derniers jours",
+			"lastNinetyDays": "90 derniers jours",
+			"lastTwelveMonths": "12 derniers mois",
+			"custom": "Personnalisé",
+			"from": "De",
+			"to": "À"
+		},
+		"compare": {
+			"lessThan": "Moins de",
+			"greaterThan": "Plus grand que",
+			"exact": "Exact",
+			"range": "Plage",
+			"lessThanLabel": "moins de {{value}}",
+			"greaterThanLabel": "plus grand que {{value}}",
+			"andLabel": "et"
+		},
+		"addFilter": "Ajouter un filtre"
+	},
+	"errorBoundary": {
+		"badRequestTitle": "400 - Requête incorrecte",
+		"badRequestMessage": "La requête n'a pas pu être interprétée par le serveur en raison d'une syntaxe incorrecte.",
+		"notFoundTitle": "404 - Il n'y a pas de page à cette adresse",
+		"notFoundMessage": "Vérifiez l'URL et réessayez, ou utilisez la barre de recherche pour trouver ce que vous cherchez.",
+		"internalServerErrorTitle": "500 - Erreur interne du serveur",
+		"internalServerErrorMessage": "Une erreur inattendue est survenue sur le serveur. Veuillez réessayer plus tard.",
+		"defaultTitle": "Une erreur est survenue",
+		"defaultMessage": "Une erreur inattendue est survenue lors du rendu de cette page.",
+		"noMatchMessage": "La page que vous recherchez n'existe pas.",
+		"backToDashboard": "Retour au tableau de bord"
+	},
+	"addresses": {
+		"shippingAddress": {
+			"header": "Adresse de livraison",
+			"editHeader": "Modifier l'adresse de livraison",
+			"editLabel": "Adresse de livraison",
+			"label": "Adresse de livraison"
+		},
+		"billingAddress": {
+			"header": "Adresse de facturation",
+			"editHeader": "Modifier l'adresse de facturation",
+			"editLabel": "Adresse de facturation",
+			"label": "Adresse de facturation",
+			"sameAsShipping": "Identique à l'adresse de livraison"
+		},
+		"contactHeading": "Contact",
+		"locationHeading": "Emplacement"
+	},
+	"email": {
+		"editHeader": "Modifier l'email",
+		"editLabel": "Email",
+		"label": "Email"
+	},
+	"transferOwnership": {
+		"header": "Transférer la propriété",
+		"label": "Transférer la propriété",
+		"details": {
+			"order": "Détails de la commande",
+			"draft": "Détails du brouillon"
+		},
+		"currentOwner": {
+			"label": "Propriétaire actuel",
+			"hint": "Le propriétaire actuel de la commande."
+		},
+		"newOwner": {
+			"label": "Nouveau propriétaire",
+			"hint": "Le nouveau propriétaire à transférer la commande à."
+		},
+		"validation": {
+			"mustBeDifferent": "Le nouveau propriétaire doit être différent du propriétaire actuel.",
+			"required": "Le nouveau propriétaire est requis."
+		}
+	},
+	"sales_channels": {
+		"availableIn": "Disponible dans <0>{{x}}</0> de <1>{{y}}</1> canaux de vente"
```

```suggestion
		"availableIn": "Disponible dans <0>{{x}}</0> canaux de vente sur <1>{{y}}</1>"
```
Wouldnt this be more accurate?